src/FlaUI.Core/Elements/Infrastructure/Element.cs

```diff
@@ -143,15 +145,37 @@ public void CaptureToFile(string filePath)
         /// </summary>
         public Element[] FindAll(TreeScope treeScope, ConditionBase condition)
         {
-            return AutomationObject.FindAll(treeScope, condition);
+            return FindAll(treeScope, condition, TimeSpan.FromMilliseconds(200));
```

Comment:
Added a static default time and set it to 1000 milliseconds. With the default retry interval of 200 milliseconds this should result in 5 retries.